core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -255,6 +255,12 @@ public enum DefaultDriverOption implements DriverOption {
    * <p>Value-type: {@link String}
    */
   SSL_KEYSTORE_PASSWORD("advanced.ssl-engine-factory.keystore-password"),
+  /**
+   * The duration between attempts to reload the keystore.
+   *
+   * <p>Value-type: {@link java.time.Duration}
+   */
+  SSL_KEYSTORE_RELOAD_INTERVAL("advanced.ssl-engine-factory.keystore-reload-interval"),
```

Comment:
it was surprising to me that the revapi checker doesn't like this:

```
[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.10.5:check (default) on project java-driver-core: The following API problems caused the build to fail:
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.ADDRESS_TRANSLATOR_CLASS: The enum constant was defined on position 61 but is now on 62. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.CLOUD_SECURE_CONNECT_BUNDLE: The enum constant was defined on position 122 but is now on 123. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.COALESCER_INTERVAL: The enum constant was defined on position 116 but is now on 117. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.COALESCER_MAX_RUNS: The enum constant was defined on position 115 but is now on 116. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.CONNECTION_CONNECT_TIMEOUT: The enum constant was defined on position 124 but is now on 125. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.CONTROL_CONNECTION_AGREEMENT_INTERVAL: The enum constant was defined on position 98 but is now on 99. The user code can break if it relies on the return value of the "ordinal()" method.
[ERROR] java.field.enumConstantOrderChanged: field com.datastax.oss.driver.api.core.config.DefaultDriverOption.CONTROL_CONNECTION_AGREEMENT_TIMEOUT: The enum constant was defined on position 99 but is now on 100. The user code can break if it relies on the return value of the "ordinal()" method.
```

I guess changing ordinal positioning could break code that depends on the ordering, although that seems pretty fragile to depend on

Reply:
Yeah I'd prefer to keep the driver options ordered into groups, like keeping all the SSL options together. I'd be surprised if anyone was serializing these options based on their enum order. I'll fix this up though - no need to rock the boat.